frame/crowdloan-rewards/Cargo.toml

```diff
@@ -32,7 +32,7 @@ libsecp256k1 = { version = "0.7.0", default-features = false, optional = true, f
     "hmac",
     "static-context",
 ] }
-balances = { package = "pallet-balances", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false, optional = true }
+balances = { package = "pallet-balances", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false, features = [ "std" ], optional = true }
```

Comment:
remove this line i guess. we do not make direct deps on this in other pallets. use NativeCurrency dep

Reply:
Are you suggesting we remove balances entirely?

Reply:
ahh its opptional

Reply:
if its used only by benchmarks, best to have it only in dev-dependencies

Reply:
This should be resolved in ed0e6f51bb61fdc2388878ec8634796d98432221. Let me know if it's not. Everything seems okay locally and CI is going well for now.